src/util/respond.js

```diff
@@ -0,0 +1,6 @@
+module.exports = ctx => {
```

Comment:
I don't think I want to extract the response. The POSt is now missing the 201 response code. Abstracting it out doesnt give as much flexibility. I think we're best off leaving it in the controller. What d o you think?

Reply:
I've been thinking about it for a while now -- going back and forth -- and every time I try to do something more complex than straight CRUD I find myself regretting that I didn't abstract the response out somewhere. lets talk sometime this week about it? i think there could be a lesser degree of abstraction -- seminara's `viewHandlers` i think still have individual statuses -- but I'm becoming less and less into the concept of sending the response from the controller.